.travis/build_computecpp.sh

```diff
@@ -5,7 +5,8 @@ set -ev
 ###########################
 # Get ComputeCpp
 ###########################
-wget --no-check-certificate https://computecpp.codeplay.com/downloads/computecpp-ce/0.5.0/ubuntu-14.04-64bit.tar.gz
+wget --no-check-certificate https://computecpp.codeplay.com/downloads/computecpp-ce/0.5.1/ubuntu-14.04-64bit.tar.gz
 tar -xzf ubuntu-14.04-64bit.tar.gz -C /tmp
+mv /tmp/ComputeCpp-CE-0.5.1-Ubuntu-14.04-64bit /tmp/computecpp
```

Comment:
What if the folder already exists?

Reply:
It won't, the machine is fresh every time.